firmware/examples/Gpio/source/main.cpp

```diff
@@ -0,0 +1,33 @@
+#include "L1_Drivers/gpio.hpp"
+
+int main(void)
+{
+    // Port 2, pin 0 must be connected to a switch that is able to ground and
```

Comment:
Can you use one of the pins on the SJTwo board that is connected to a Button and LED? Then put in a note, saying that pressing the onboard button should light up the led.